Lombiq.Tests.UI/Extensions/NavigationUITestContextExtensions.cs

```diff
@@ -271,5 +277,7 @@ public static void DismissAlert(this UITestContext context)
         context.GoToRelativeUrlAsync("/Contents/ContentItems/" + contentItemId);
 
     public static Task GoToContentItemEditorByIdAsync(this UITestContext context, string contentItemId) =>
-        context.GoToRelativeUrlAsync($"/Admin/Contents/ContentItems/{contentItemId}/Edit");
+        string.IsNullOrEmpty(context.CustomAdminUrl)
+            ? context.GoToRelativeUrlAsync($"/Admin/Contents/ContentItems/{contentItemId}/Edit")
+            : context.GoToRelativeUrlAsync($"/{context.CustomAdminUrl}/Contents/ContentItems/{contentItemId}/Edit");
```

Comment:
Instead of having this if-else here and similar ones in `OrchardCoreDashboardUITestContextExtensions`, add a `GoToAdminRelativeUrlAsync` and handle this prefixing there.

Reply:
So basically instead of else-if-ing just created another method everywhere with RelativeUrl in the method name and pass the custom url?

Reply:
No, pass $"/Contents/ContentItems/{contentItemId}/Edit" to `GoToAdminRelativeUrlAsync` and only access `CustomAdminUrl` in the latter.

Reply:
That is basically a new method instead of case dividing or not? 
I am asking this because if I create new extension methods like GoToContentItemPageRelativeAsync() because you suggested to leave the IF-ELSE syntax note that there are several other extension methods that are using these basic ones. So if I create a GoToAdminRelativeUrlAsync instead of GoToDashboardAsync with if-else that means that I have to address every single occurence of every method and case-divide there.

For example TestContentOperationsAsync() is using GoToContentItemsPageAsync() now it is perfect because if there is a CUstomUrl it will use the custom if not then /Admin no problem. But I remove if-else and there will be a GoToContentItemsRelative() then I have to case divide in these methods. Probably you understand it I just tried to make it absolutely clear what I am thinking and ask you what do you think about this.

Reply:
Or I am the one that didn't catch your line of thoughts is also a possibility :D

Reply:
It's about adding a new method, yes. Instead of having to do this if-else everywhere, you do it once (and not even once, see my remark under `CustomAdminUrl`). But I'm talking about adding a new method while keeping `GoToDashboardAsync` and the others still, just not copy-paste this `string.IsNullOrEmpty(context.CustomAdminUrl) ? ... : ...` everywhere.

Reply:
What if we don't create a single new method just give basic value to the AdminPrefix as you suggested and after that we just use the GoToPage<T>(string relativeUrl) overload everywhere?

Reply:
The issue with that is that you have to remember to reference and use `CustomAdminUrl` everywhere, instead of a dedicated method making it obvious. And if we want to change how such admin URLs are created and navigated to (what we're just doing) then we'd need to do it in every place instead of one. I.e., DRY.